src/data/types/capital-types.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  AuthorityType,
+  LoanProgramStatus,
+  ProjectType,
+} from '@rewiringamerica/capital-common';
+import { FromSchema } from 'json-schema-to-ts';
+
+/**
+ * Schema for Image.
+ */
+export const IMAGE_SCHEMA = {
+  $id: 'Image.schema.json',
+  type: 'object',
+  properties: {
+    src: { type: 'string' },
+  },
+  required: ['src'],
+  additionalProperties: false,
+} as const;
```

Comment:
There's already an image schema in `src/schemas/v1/image.ts` (which admittedly should be in `src/data` somewhere instead). We should use that instead of defining a duplicate one.

That other schema has width and height as required properties, so you would need to add those to whatever data source you're going to use for this capital data. (I think it's correct that they're required; it improves rendering on the web if image dimensions are known statically.)